modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -3012,7 +3012,8 @@ namespace fisheye
         CALIB_FIX_K3                = 1 << 6,
         CALIB_FIX_K4                = 1 << 7,
         CALIB_FIX_INTRINSIC         = 1 << 8,
-        CALIB_FIX_PRINCIPAL_POINT   = 1 << 9
+        CALIB_FIX_PRINCIPAL_POINT   = 1 << 9,
+        CALIB_ZERO_DISPARITY        = 0x00400
```

Comment:
Fixes issue where fisheye::CALIB_ZERO_DISPARITY does not exit

Reply:
> = 0x00400

`= cv::CALIB_ZERO_DISPARITY`

to ensure that the same value is used anywhere (implementation still uses `cv::CALIB_ZERO_DISPARITY` internally).

Reply:
That shows up weird in the docs. 

Just testing this before pushing:
`TEST_F(fisheyeTest, stereoRectify)`
`{`
`    // For consistency puproses `
 `   CV_Assert(cv::CALIB_ZERO_DISPARITY == cv::fisheye::CALIB_ZERO_DISPARITY)`

Reply:
I believe check is ok here. But please use `CV_StaticAssert()` for that (in C++11 it is compilation error)